### PR TITLE
fix link to annotation in summary page

### PR DESCRIPTION
**Pull request by TomWoodward**

### CODE REVIEW

**Comment by TomWoodward** [comment]
[Bug: View in book link on summary page doesn't scroll to correct annotation](https://trello.com/c/PgJSK7xH/1587-bug-view-in-book-link-on-summary-page-doesnt-scroll-to-correct-annotation)

File: tutor/src/components/annotations/annotation-card.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export default class AnnotationCard extends React.Component {
   @autobind
   openPage() {
     const { id, selection: { courseId, chapter, section } } = this.props.annotation;
-    let url = `/books/${courseId}/section/${chapter}`;
+    let url = `/book/${courseId}/section/${chapter}`;
     if (section) {
```
Comment (nathanstitt) [inline tutor/src/components/annotations/annotation-card.jsx]: sigh, yeah we renamed that url awhile ago and thought we'd updated all the links.  @TomWoodward can you search through the code and see if you spot any other `books/` urls?

Comment (TomWoodward) [reply tutor/src/components/annotations/annotation-card.jsx]: @nathanstitt does this count https://github.com/openstax/tutor-js/blob/master/tutor/src/screens/new-course/bio2e-unavailable.jsx#L34 ?

Comment (nathanstitt) [reply tutor/src/components/annotations/annotation-card.jsx]: nah that one's to CNX, and does appear to be correct.  Thanks for checking!

**Pull request merged**